src/Cohesity.Powershell/Scripts/View/Add-CohesityViewWhitelist.ps1

```diff
@@ -0,0 +1,91 @@
+function Add-CohesityViewWhitelist {
+    <#
+        .SYNOPSIS
+        Add whitelist IP(s) for a given view.
+        .DESCRIPTION
+        Add whitelist IP(s) for a given view.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Add-CohesityViewWhitelist -IP4List "1.1.1.1", "2.2.2.2" -NetmaskIP4 "255.255.255.0"
+        Add whitelist IP(s) an override global whitelist for a given view.
+        .EXAMPLE
+        Add-CohesityViewWhitelist -IP4List "1.1.1.1", "2.2.2.2" -NetmaskIP4 "255.255.255.0" -NFSRootSquash -NFSAccess "kReadWrite" -NFSAllSquash -SMBAccess "kReadWrite"
+        Add whitelist IP(s) an override global whitelist for a given view with optional parameters
```

Comment:
We have started a project to have inclusive names. Can you please change whitelist to AllowList everywhere ? 

If the cmdlet is already released. lets introduce a new param/cmdlet and mark the older one deprecated. Give it 6 months and then deprecate the older one.

Reply:
We do not have any param/cmdlet with non-inclusive items. Will avoid using non-inclusive names for the future cmdlet/params.